seedDB/README.md

```diff
@@ -4,7 +4,31 @@ These files are MySQL database dump files for seeding a new instance of the cBio
 
 The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database schema 2.1.0. This means that the version numbers won't be identical. cBioPortal 1.9.0 with database schema 2.4.0 removed PDB annotations from the database.
 
-## Latest seed database
+Instructions for building and updating seedDBs [HERE](#for-developers)
+
+## Release Notes
+### Latest seed database
+
+#### Seed database schema 2.12.8
+
+This schema is required for cBioPortal release versions:
+- **2.0.0**
+
+When using a release version **> 2.0.0**, a migration step to a new database schema might be required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
+
+**Schema 2.12.8**: [SQL file with create table statements](https://raw.githubusercontent.com/cBioPortal/cbioportal/v3.6.10/db-scripts/src/main/resources/cgds.sql)<br>
+**Seed database**: [seed-cbioportal_hg19_v2.7.3.sql.gz](https://github.com/cBioPortal/datahub/raw/master/seedDB/seed-cbioportal_hg19_v2.7.3.sql.gz)<br>
```

Comment:
I will need to give you the new seed database once we have finalised it.

Reply:
And the md5sum for it.

Reply:
Thanks @mandawilson added the sql, updated the link and md5checksum.